bridge360ph/wayste-household-app
# Improved UI; Test build 1

# Overview

- Improved general UI and appearance and refactored modules

# QA Testing

- Run locally on browser (desktop) -- See demo
- Build for android -- (I'm currently having issues on this one)

# Demo

https://www.loom.com/share/db0ac016f9e44c62a50838d99968627f
